modules/services/home-manager-auto-upgrade.nix

```
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.services.home-manager.auto-upgrade;
+
+in {
+  meta.maintainers = [ lib.maintainers.pinage404 ];
+
+  options = {
+    services.home-manager.auto-upgrade = {
+      enable = lib.mkEnableOption ''
+        Home Manager upgrade service that periodically updates your nix channels before
+        running `home-manager switch`
+      '';
```

I think you need to put the `''` on the same line so that the `.` isn't on the next line in the rendered docs.

Also, use `<code>` (I think?), maybe it should be `<literal>`? to surround the command text.

Take all this with a grain of salt, and check the documentation on your local machine to make sure.

```suggestion
        running <code>home-manager switch</code>'';
```

---

`<code>` and `<literal>` work and look like the same